<ui version="4.0" >
 <class>ds_genClass</class>
 <widget class="QMainWindow" name="ds_genClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize" >
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="leftMargin" >
     <number>1</number>
    </property>
    <property name="topMargin" >
     <number>1</number>
    </property>
    <property name="rightMargin" >
     <number>1</number>
    </property>
    <property name="bottomMargin" >
     <number>1</number>
    </property>
    <item>
     <widget class="QTabWidget" name="mainTabWidget" >
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Задание динамической системы</string>
       </attribute>
       <layout class="QGridLayout" >
        <item rowspan="2" row="0" column="0" >
         <widget class="QGroupBox" name="groupBox_5" >
          <property name="title" >
           <string>Решение ДУ</string>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
           <item row="0" column="0" colspan="2" >
            <widget class="QComboBox" name="methodCBox" >
             <item>
              <property name="text" >
               <string>Метод Эйлера</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Метод Рунге-Кутты</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_17" >
             <property name="text" >
              <string>Шаг интегрирования</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QDoubleSpinBox" name="intstepSBox" >
             <property name="decimals" >
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_24" >
             <property name="text" >
              <string>Уровень шума</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QDoubleSpinBox" name="noise_lvlSBox" >
             <property name="decimals" >
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item rowspan="5" row="0" column="1" >
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="title" >
           <string>Начальные значения</string>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
           <item row="0" column="0" >
            <widget class="QCheckBox" name="conststartvaluesBox" >
             <property name="text" >
              <string>Удерж. пост. на каждом шаге</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QGroupBox" name="groupBox_9" >
          <property name="title" >
           <string/>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_26" >
             <property name="text" >
              <string>Кол-во уравнений</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QSpinBox" name="eqnsSBox" />
           </item>
          </layout>
         </widget>
        </item>
        <item rowspan="4" row="1" column="2" >
         <widget class="QTextEdit" name="dssummaryTEdit" >
          <property name="readOnly" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QGroupBox" name="groupBox_6" >
          <property name="title" >
           <string>Настройка временного ряда</string>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
           <item row="0" column="0" >
            <widget class="QComboBox" name="timeseriestypeCBox" >
             <item>
              <property name="text" >
               <string>Длина ряда</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Кол-во точек</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="label_27" >
             <property name="text" >
              <string>до</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="label_28" >
             <property name="text" >
              <string>шаг</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QSpinBox" name="serieslengthsSBox" >
             <property name="maximum" >
              <number>100000000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QSpinBox" name="serieslengtheSBox" >
             <property name="maximum" >
              <number>100000000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QSpinBox" name="serieslengthstepSBox" >
             <property name="maximum" >
              <number>100000000</number>
             </property>
             <property name="singleStep" >
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_19" >
             <property name="text" >
              <string>Шаг выборки от</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="label_18" >
             <property name="text" >
              <string>до</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2" >
            <widget class="QLabel" name="label_25" >
             <property name="text" >
              <string>шаг</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QSpinBox" name="seriesstepsSBox" >
             <property name="maximum" >
              <number>200</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QSpinBox" name="seriesstepeSBox" />
           </item>
           <item row="3" column="2" >
            <widget class="QSpinBox" name="seriesstepstepSBox" />
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QGroupBox" name="groupBox_4" >
          <property name="title" >
           <string>Связь</string>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_12" >
             <property name="text" >
              <string>Сила связи от</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="label_22" >
             <property name="text" >
              <string>до</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="label_23" >
             <property name="text" >
              <string>шаг</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QDoubleSpinBox" name="forcecouplbSBox" >
             <property name="decimals" >
              <number>3</number>
             </property>
             <property name="minimum" >
              <double>-99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QDoubleSpinBox" name="forcecoupleSBox" >
             <property name="decimals" >
              <number>3</number>
             </property>
             <property name="minimum" >
              <double>-99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QDoubleSpinBox" name="forcecouplstepSBox" >
             <property name="decimals" >
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_13" >
             <property name="text" >
              <string>Ведомое</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2" >
            <widget class="QComboBox" name="responseCBox" >
             <item>
              <property name="text" >
               <string>x1</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x2</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x3</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x4</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x5</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x6</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_14" >
             <property name="text" >
              <string>Ведущее</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2" >
            <widget class="QComboBox" name="driverCBox" >
             <item>
              <property name="text" >
               <string>x1</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x2</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x3</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x4</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x5</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>x6</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QLabel" name="label_15" >
             <property name="text" >
              <string>Тип связи</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2" >
            <widget class="QComboBox" name="coupltypeCBox" >
             <item>
              <property name="text" >
               <string>Пропорциональная</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Разностная</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QGroupBox" name="groupBox_7" >
          <property name="title" >
           <string>Сохранение данных</string>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_20" >
             <property name="text" >
              <string>Папка</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QPushButton" name="folderchooseButt" >
             <property name="text" >
              <string>Выбор...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_21" >
             <property name="text" >
              <string>Тип данных</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QComboBox" name="datatypeCBox" >
             <item>
              <property name="text" >
               <string>Сжатые Qt</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Текст</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0" colspan="3" >
         <widget class="QPushButton" name="generateButt" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Генерировать!</string>
          </property>
          <property name="autoDefault" >
           <bool>true</bool>
          </property>
          <property name="default" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2" >
         <widget class="QProgressBar" name="progressBar" >
          <property name="value" >
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="6" column="2" >
         <widget class="QLabel" name="elapsedTime" >
          <property name="text" >
           <string>Выполнено за: </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Визуализация</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QGroupBox" name="groupBox_8" >
          <property name="title" >
           <string>какие переменные строить</string>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>1</number>
           </property>
           <property name="topMargin" >
            <number>1</number>
           </property>
           <property name="rightMargin" >
            <number>1</number>
           </property>
           <property name="bottomMargin" >
            <number>1</number>
           </property>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="drawGraphButt" >
          <property name="text" >
           <string>Построить график</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu" >
    <property name="title" >
     <string>Файл</string>
    </property>
    <addaction name="action" />
    <addaction name="action_2" />
    <addaction name="action_3" />
   </widget>
   <addaction name="menu" />
  </widget>
  <action name="action" >
   <property name="text" >
    <string>Открыть профиль...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2" >
   <property name="text" >
    <string>Сохранить профиль...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_3" >
   <property name="text" >
    <string>Создать профиль</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="1" margin="1" />
 <resources/>
 <connections/>
</ui>
